shapeworks_cloud/core/templates/index.html

```diff
@@ -1,13 +0,0 @@
-{% extends "base.html" %}
-
-{% block title %}
-Welcome
-{% endblock %}
-
-{% block content %}
-{% if user.is_authenticated %}
-Web UI coming soon.  Please see the <a href="https://github.com/girder/shapeworks-cloud/blob/master/swcc/usage.ipynb">python client</a>.
-{% else %}
-If you don't have an account, click <a href="{% url 'account_signup' %}">here</a> to register.
-{% endif %}
-{% endblock %}
```

Comment:
I think this index page is worth keeping around. It's much more informative than a blank 404 page.

Reply:
Ok, I put it back in [aabc59b](https://github.com/girder/shapeworks-cloud/pull/130/commits/aabc59b6b1a2935cd83e486cc61df359711682a2)